.babelrc

```diff
@@ -1,6 +1,5 @@
 {
   "presets": [
-    "es2015",
```

Comment:
I wanted to upgrade things to the support the preset used by facebook incubator, it has 27k+ start, so I think it's becoming very popular. 

You can read more on it here: 
https://github.com/facebookincubator/create-react-app/tree/master/packages/babel-preset-react-app

Now looking at it, I think it should read this instead of the words preset,
```
{
  "presets": [
    "react-app"
  ]
}```

I'll look into this and update.